lib/msal-browser/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log - @azure/msal-browser
```

Comment:
Hmm this isn't right, looks like it isn't account for the existing `changelog.md` file.

Reply:
Guessing this is beachball's fault, maybe we need to make sure all existing change logs are `CHANGELOG.md` not `changelog.md`.

Reply:
Dang, we were having this issue locally as well and couldn't figure it out. That sounds like the right solution if they changed the casing

Reply:
Yes, that sounds like the correct solution. In fact there's places they hard code 'CHANGELOG' as the expected name so probably better to use the uppercase version. Will be opening a PR to address this.